### PR TITLE
manage namespace projects with rows

### PR DESCRIPTION
**Pull request by grosser**
maybe simpler than multi-select 🤷‍♂ 
Screen Shot 2019-05-22 at 1 15 04 PM

@zendesk/compute

### CODE REVIEW

File: plugins/kubernetes/app/views/kubernetes/namespaces/_form.html.erb [hunk 1]
```diff
@@ -8,8 +8,14 @@
...
-      <%= form.input :project_ids, label: 'Projects', help: "Adding a projects will remove it's kubernetes roles resource and service names AND move it from it's current namespace" do %>
-        <%= form.select :project_ids, Project.pluck(:name, :id), {}, multiple: true, **Samson::FormBuilder::LIVE_SELECT_OPTIONS %>
+      <%= form.input :projects, help: "Adding a projects will remove it's kubernetes roles resource and service names AND move it from it's current namespace" do %>
+        <% projects = Project.pluck(:name, :id) %>
+        <% (@kubernetes_namespace.projects + [Project.new]).each do |project| %>
+          <div>
+            <%= live_select_tag "kubernetes_namespace[project_ids][]", options_for_select(projects, project.id), include_blank: true %>
+          </div>
+        <% end %>
+        <%= link_to "Add row", "#", class: "duplicate_previous_row" %>
```
Comment (adammw) [inline plugins/kubernetes/app/views/kubernetes/namespaces/_form.html.erb]: auto add a blank row?

Comment (grosser) [reply plugins/kubernetes/app/views/kubernetes/namespaces/_form.html.erb]: that's what it does

Comment (adammw) [reply plugins/kubernetes/app/views/kubernetes/namespaces/_form.html.erb]: not sure why you need the link then? 😕

Comment (grosser) [reply plugins/kubernetes/app/views/kubernetes/namespaces/_form.html.erb]: to add multiple projects at once

**Review by adammw — APPROVED** [review]

**Comment by adammw** [comment]
:shrug:

**Pull request merged**